Source/WebCore/html/ImageDocument.cpp

```diff
@@ -231,6 +231,7 @@ void ImageDocument::createDocumentStructure()
 
     auto body = HTMLBodyElement::create(*this);
     body->setAttribute(styleAttr, "margin: 0px"_s);
+    rootElement->setAttribute(styleAttr, "color-scheme: dark light"_s);
```

Comment:
Nit: Can we use `light dark` instead, to match the other instances of this string in the project?